sdk/iOS/src/MSPullSettings.h

```diff
@@ -0,0 +1,20 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ----------------------------------------------------------------------------
+
+#import <Foundation/Foundation.h>
+
+/// Settings that control the pull behavior
+@interface MSPullSettings : NSObject
+
+#pragma mark * Properties
+
+///@name Properties that control pull behavior
+///@{
+
+/// Size of pages requested from the server while performing a pull
+@property (nonatomic) NSInteger pageSize;
+
+///@}
+
```

Comment:
It'd be good to make that change (and UTs for it) separately. It might be better to just let the user specify the NSQualityOfService instead of inferring it from the source (user initiated, etc) of the task.